src/main/java/huan/main/Huan.java

```diff
@@ -0,0 +1,142 @@
+package huan.main;
+
+import huan.task.*;
+
+import java.util.Scanner;
+import java.util.ArrayList;
+import java.util.List;
+public class Huan {
+    private static List<Task> tasks = new ArrayList<>();
+
+    public static Boolean isIndexValid(int index) {
+        return index >= 1 && index <= tasks.size();
+    }
+    public static void addTask(Task newTask) {
+        tasks.add(newTask);
+    }
+
+    public static void listTasks() {
+        int cnt = 0;
+        System.out.println("You have a total of " + tasks.size() + " tasks.");
+        for (Task task : tasks) {
+            cnt += 1;
+            System.out.printf(cnt + ". ");
+
+            switch (task.getTaskType()) {
+            case (1):
+                TodoTask todoTask = (TodoTask)task;
+                todoTask.printTask();
+                break;
+            case (2):
+                EventTask eventTask = (EventTask)task;
+                eventTask.printTask();
+                break;
+            case (3):
+                DeadlineTask deadlineTask = (DeadlineTask)task;
+                deadlineTask.printTask();
+                break;
+            default:
+                task.printTask();
+                break;
+            }
+        }
+    }
+    public static void readProcessCommand() {
+
+        Scanner scanner = new Scanner(System.in);
+        while(true) {
+            System.out.println("-------------------------");
+            String inputCommand = scanner.nextLine();
+            String[] words = inputCommand.split(" ");
+            String firstWord = words[0];
+            String suffixWord;
+            if(words.length > 1) {
+                suffixWord = inputCommand.substring(words[0].length() + 1);
```

Comment:
Multiple usage of Magic numbers, try to assign the number with a meaningful variable name for easy understand of what the number is for.